components/WorkoutList/WorkoutList.js

```diff
@@ -18,21 +23,96 @@ export default function WorkoutList() {
     }
   }
 
-  return (<>
+
+  return (<CardContainer>
     {data.map(workout => (
-      <div key={workout._id}>
-        <h2>{workout.name}</h2>
-        <ul>
-          {workout.exercises.map(({ exercise, sets, reps }) => (
-            <li key={exercise._id}>
-              <p>{exercise.name}</p>
-              <p>Sets: {sets}</p>
-              <p>Reps: {reps}</p>
-            </li>
+      <Card key={workout._id}>
+        <ListContainer>
+          <h2>{workout.name}</h2>
+          {workout.exercises.map(({ exercise, sets, reps, weight }, index) => (<Fragment key={exercise._id}>
+            <ListItem>
+              <Textwrapper>
+                <Text>{exercise.name}</Text>
+                {weight && <Text style={{ color: 'grey', fontSize: '' }}>{exercise.difficulty}</Text>}
+              </Textwrapper>
+              <Textwrapper>
+                <Text>{sets} x {reps}</Text>
+                {weight && <Text style={{ color: 'grey', fontSize: '' }}>{weight}kg</Text>}
```

Comment:
Instead of working with inline-styles, please make use of a prop for your `Text` component